test/bats/azure.bats

```diff
@@ -6,13 +6,16 @@ BATS_TESTS_DIR=test/bats/tests
 WAIT_TIME=60
 SLEEP_TIME=1
 IMAGE_TAG=v0.0.8-e2e-$(git rev-parse --short HEAD)
-NAMESPACE=dev
+NAMESPACE=default
 PROVIDER_YAML=https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/deployment/provider-azure-installer.yaml
 
-export SECRET_NAME=secret1
-export KEY_NAME=key1
-export SECRET_NAME=secret1
-export SECRET_VERSION=""
+export KEYVAULT_NAME=${KEYVAULT_NAME:-csi-secrets-store-e2e}
+export SECRET_NAME=${KEYVAULT_SECRET_NAME:-secret1}
+export SECRET_VERSION=${KEYVAULT_SECRET_VERSION:-""}
+export SECRET_VALUE=${KEYVAULT_SECRET_VALUE:-"test"}
+export KEY_NAME=${KEYVAULT_KEY_NAME:-key1}
+export KEY_VERSION=${KEY_VERSION:-7cc095105411491b84fe1b92ebbcf01a}
```

Comment:
nit: to be consistent, shouldnt this be `KEYVAULT_KEY_VERSION`

Reply:
updated!